azext_edge/edge/_help.py

```diff
@@ -1953,3 +1956,33 @@ def load_iotops_help():
           text: >
             az iot ops schema show-dataflow-refs --registry myregistry -g myresourcegroup --latest
     """
+
+    helps[
+        "iot ops clone"
+    ] = f"""
+        type: command
+        short-summary: Clone an instance.
+        long-summary: |
+          Clone analyzes an instance then reproduces it in an infrastructure-as-code
+          manner via ARM templates. The output of clone may be applied directly to another connected cluster
+          or, be saved locally to use at another time (potentially with modification).
+
+          Clone is compatible with the following instance version range: {COMPAT_INSTANCE_VERS_MIN}>=,<{COMPAT_INSTANCE_VERS_MAX}
+
+        examples:
+        - name: Clone an instance to a desired connected cluster.
+          text: >
+            az iot ops clone -n myinstance -g myresourcegroup --to-cluster-id $CLUSTER_RESOURCE_ID
+        - name: Similar to the prior example, but splitting and serially applying asset related sub-deployments.
```

Comment:
Might be better to not have these help descriptions refer to previous examples - what if an AI tool picks out one of these examples without displaying the above one or shuffles the order?